app/helpers/tasks/collection_objects/area_and_date/area_and_date_helper.rb

```diff
@@ -17,4 +17,22 @@ def id_range_select_tag(id_range_element, option_list)
     select_tag(id_range_element, options_for_select(option_list), prompt: 'Select an identifier')
   end
 
+  def get_first_created_at
+    colobj = CollectionObject.with_project_id(sessions_current_project_id).first_created
+    if colobj.blank?
+      '1700/01/01'
+    else
+      colobj.created_at.to_date.to_s.gsub('-', '/')
+    end
+  end
+
+  def get_last_created_at
+    colobj = CollectionObject.with_project_id(sessions_current_project_id).last_created
+    if colobj.blank?
+      Date.today.to_s.gsub('-', '/')
```

Comment:
For future consideration: https://stackoverflow.com/questions/41965408/rails-4-erb-consistent-date-format

Reply:
Do you want the default date format for the system set to `yyyy/mm/dd`?